content/millennium/r4/conformance/capability-statement.md

```diff
@@ -31,7 +31,7 @@ Authorization is not required.
 
 ### Headers
 
-<%= headers head: {'<a href="../#media-types">Accept</a>': 'application/fhir+json'} %>
+<%= headers head: {Accept: 'application/json+fhir'} %>
```

Comment:
this is R4 and should be fhir+json

Reply:
Made the change here: [Fix](https://github.com/cerner/fhir.cerner.com/pull/323/commits/7a5c7155c88812a895674b0212f9b08f67a89fa7#diff-eb7872b6d7136d8c355e3b7d31d6feaaL34)